show-callout/README.md

```diff
@@ -16,9 +16,7 @@ Tap anywhere on the map. A callout showing the WGS84 coordinates for the tapped
 
 1. When the user taps, get the tapped location(map point) from the `SingleTapConfirmedEvent`.
 2. Project the point's geometry to WGS84 using `GeometryEngine.projectOrNull(mapPoint, SpatialReference.wgs84())`.
-3. Create a new Android TextView object and set its text to the coordinate string from the point.
```

Comment:
I think you also need a step for adding the callout to the MapView content, passing in the location and callout content

Reply:
I've reworded and broken into more detailed steps. Thanks!